build.gradle

```diff
@@ -34,7 +34,7 @@ java {
 }
 
 javafx {
-    version = "17"
+    version = "19-ea+10"
```

Comment:
Please revert this, I don't want this to use an alpha release

Reply:
Well, I needed it to test it against jfx19 which is not yet released to identify the name clashes which is the whole point of this PR. Changed it now to 18